### ISSUE TITLE
[Major Bug] Skin broke

### ISSUE THREAD

**Issue by BoB-Client** [Thy-Vipe/BeastsOfBermuda-issues#3700]
_Originally written by **Cleora | 76561198391039913**_
Game Version: 1.1.1124

*===== System Specs =====
CPU Brand: Intel(R) Core(TM) i5-9600K CPU @ 3.70GHz
Vendor: GenuineIntel
GPU Brand: NVIDIA GeForce RTX 2070 SUPER
GPU Driver Info: Unknown
Num CPU Cores: 6
===================*
Context: **It happened while changing skins, then i moved forward and roared and got glitched** 
Map: Rival_Shores

Basically with the new skin update, we got the option to change skins, I applied a new pattern and put two things as pink and one as brown then got kicked or glitched out of the menu and now im forever looking like a ugly pink turd even after putting in 3+ months to get it 2.0 almost 2.1+ in size on Le Hideout server

**Comment by Trip-Trap** [comment]
:X wellllllllllll
maybe need confirmation rather than just closing the o menu?
